satnogsclient/observer/worker.py

```diff
@@ -50,6 +52,8 @@ def __init__(self, ip, port, time_to_stop=None, frequency=None, proc=None):
             self._observation_end = time_to_stop
         if proc:
             self._gnu_proc = proc
+        if _post_exec_script:
```

Comment:
If `_post_exec_script` is anything that evaluates to `False`, the assignment won't take place.